test/integration/auth.spec.ts

```diff
@@ -1301,29 +1305,68 @@ describe('admin.auth', () => {
   describe('Tenant management operations', () => {
     let createdTenantId: string;
     const createdTenants: string[] = [];
+    const mfaSmsEnabledTotpEnabledConfig: MultiFactorConfig = {
+      state: 'ENABLED',
+      factorIds: ['phone'],
+      providerConfigs: [
+        {
+          state: 'ENABLED',
+          totpProviderConfig: {
+            adjacentIntervals: 5,
+          },
+        },
+      ],
+    };
+    const mfaSmsEnabledTotpDisabledConfig: MultiFactorConfig = {
+      state: 'ENABLED',
+      factorIds: ['phone'],
+      providerConfigs: [
+        {
+          state: 'DISABLED',
+          totpProviderConfig: {},
+        }
+      ],
+    }
+    const mfaSmsDisabledTotpEnabledConfig: MultiFactorConfig = {
+      state: 'DISABLED',
+      factorIds: [],
+      providerConfigs: [
+        {
+          state: 'ENABLED',
+          totpProviderConfig: {},
+        }
+      ],
+    }
+    const passwordConfig: PasswordPolicyConfig = {
+      enforcementState: 'ENFORCE',
+      forceUpgradeOnSignin: true,
+      constraints: {
+        requireUppercase: true,
+        requireLowercase: true,
+        requireNonAlphanumeric: true,
+        requireNumeric: true,
+        minLength: 8,
+        maxLength: 30,
+      },
+    }
+    const smsRegionAllowByDefaultConfig: SmsRegionConfig = {
+      allowByDefault: {
+        disallowedRegions: ['AC', 'AD'],
+      }
+    }
     const tenantOptions: CreateTenantRequest = {
       displayName: 'testTenant1',
       emailSignInConfig: {
         enabled: true,
         passwordRequired: true,
       },
-      multiFactorConfig: {
-        state: 'ENABLED',
-        factorIds: ['phone'],
-        providerConfigs: [
-          {
-            state: 'ENABLED',
-            totpProviderConfig: {
-              adjacentIntervals: 5,
-            },
-          },
-        ],
-      },
+      multiFactorConfig: mfaSmsEnabledTotpEnabledConfig,
       // Add random phone number / code pairs.
       testPhoneNumbers: {
         '+16505551234': '019287',
         '+16505550676': '985235',
       },
+      passwordPolicyConfig: passwordConfig,
```

Comment:
It seems like setting `passwordPolicyConfig ` here modifies our integration test project affecting the existing tests for auth. Can we move this into a separate test where we enable password config, run the tests, and disable the config in ` after(() => {` ?